docs/users-guide/10-metabot.md

```diff
@@ -0,0 +1,44 @@
+# Getting answers in Slack with Metabot
+
+You can already send data to Slack on  a set schedule via [Pulses](http://www.metabase.com/docs/latest/users-guide/07-pulses) but what about when you need an answer right now?. Say hello to Metabot.
+
+Metabot helps add context to conversations you’re having in Slack by letting you see results from Metabase as part of the conversation.
```

Comment:
Consider changing to, "Metabot helps add context to conversations you’re having in Slack by letting you insert results from Metabase."